docker/Dockerfile

```
@@ -139,6 +139,9 @@ RUN apt-get update && \
 ARG CHECKPOINT_SYNC=true
 ARG NETWORK=Mainnet
 
+ARG ZEBRA_CONF_PATH=/etc/zebrad/zebrad.toml
```

If we want all the Zebra configs to be in the same directory, we might want to use:
```suggestion
ARG ZEBRA_CONF_PATH=/etc/zebra/zebrad.toml
```

For example, we might add configs for separate tools, like the checkpoints tool, or a future RPC or raw database access tool.

This also matches the name of the git repository.